lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -93,21 +104,14 @@ class FullPageScreenshot extends FRGatherer {
 
     // Now that the viewport is taller, give the page some time to fetch new resources that
     // are now in view.
-    const networkMonitor = new NetworkMonitor(context.driver.defaultSession);
-    const waitForNetworkIdleResult = waitForNetworkIdle(session, networkMonitor, {
-      networkQuietThresholdMs: 1000,
-      busyEvent: 'network-2-busy',
-      idleEvent: 'network-2-idle',
-      isIdle: recorder => recorder.is2Idle(),
-    });
     await Promise.race([
-      new Promise(resolve => setTimeout(resolve, 1000 * 10)),
+      new Promise(resolve => setTimeout(resolve, 1000 * 5)),
       waitForNetworkIdleResult.promise,
     ]);
     waitForNetworkIdleResult.cancel();
 
     // Now that new resources are (probably) fetched, wait long enough for a layout.
-    await context.driver.executionContext.evaluate(() => {
+    await context.driver.executionContext.evaluate(function waitForDoubleRaf() {
```

Comment:
If it's going to have a name, can we put it on page functions or move it to the top of the file